chemclipse/plugins/org.eclipse.chemclipse.orbitdeps/build.properties

```diff
@@ -1,5 +1,10 @@
-source.. = src/
 output.. = bin/
 bin.includes = META-INF/,\
-               .,\
-               libraries/
+               libraries/ejml-cdense-0.38.jar,\
```

Comment:
'.' entry was deleted please make sure it is present, see previous version line 4